interface/patient_file/merge_patients.php

```diff
@@ -313,27 +483,49 @@ function mergeRows($tblname, $colname, $source_pid, $target_pid)
    <input type='hidden' name='form_source_pid' value='0' />
   </td>
   <td>
-    <?php echo xlt('This is the chart that is to be merged into the main chart and then deleted.'); ?>
+    <?php echo xlt(
+        'This is the chart that is to be merged into the main chart 
+        and then deleted.'
+    ); ?>
   </td>
  </tr>
 </table>
-<p><input type='submit' class="btn btn-primary" name='form_submit' value='<?php echo xla('Merge'); ?>' /></p>
+<p><input type='submit' class="btn btn-primary" name='form_submit' 
+    value='<?php echo xla('Merge'); ?>' /></p>
 </div>
 </form>
 <div class="jumbotron">
-    <p class="font-weight-bold"><?php echo xlt('This utility is experimental. Back up your database and documents before using it!'); ?></p>
+    <p class="font-weight-bold"><?php echo xlt(
+        'This utility is experimental. 
+    Back up your database and documents before using it!'
```

Comment:
ensure don't break up translation constant into multiple lines. need to make these one line no matter how long they are (or else will break in the translation pipeline)